src/components/shared/ReactFlow/FlowSidebar/sections/GraphComponents.tsx

```diff
@@ -97,55 +91,59 @@ const GraphComponents = ({ isOpen }: { isOpen: boolean }) => {
       userComponentsFolder.components.length > 0;
 
     return (
-      <BlockStack>
-        {hasUsedComponents && (
-          <FolderItem
-            key="used-components-folder"
-            folder={usedComponentsFolder}
-            icon={LayoutGrid}
-          />
-        )}
-        {hasFavouriteComponents && (
+      <BlockStack gap="2">
+        <UpgradeAvailableAlertBox />
```

Comment:
saves you from having to check again within the component


```suggestion
        {remoteComponentLibrarySearchEnabled  && <UpgradeAvailableAlertBox />}
```